packages/f-checkout/src/components/Checkout.vue

```diff
@@ -22,6 +22,14 @@
                 type="post"
                 :class="$style['c-checkout-form']"
                 @submit.prevent="onFormSubmit">
+                <p
```

Comment:
I feel using a p tag as a clickable element is a bit strange here? Could we use just the anchor here? Would help with tabbing or keyboard accessibility too (around the click event trigger)